### DISCUSSION TITLE
Where to get the latest source code for Samples ToDoList APP

### DISCUSSION THREAD

**Discussion by xky0007** [AvaloniaUI/Avalonia#15983] [General]
Where to get the latest source code for Samples [ToDoAPP](https://docs.avaloniaui.net/docs/tutorials/samples) from the Avalonia UI website?

Thanks.

**Comment by timunie** [comment]
Here is the new sample which is preferred as of today. Docs needs to be fixed in the next few days https://github.com/AvaloniaUI/Avalonia.Samples/tree/main/src%2FAvalonia.Samples%2FCompleteApps%2FSimpleToDoList

**Reply by xky0007** [reply]
HI, it seems this sample doesn't include navigate view topics. I want to find a sample about navigate view or window. Thanks.

**Reply by timunie** [reply]
For this please see the other samples in Avalonia.Samples, Routing sample and ViewInteraction samples namely. The way we did it in the older ToDoList is probably a bit outdated I'd say.

**Reply by thevortexcloud** [reply]
Also if you are using ReactiveUI there is existing docs for that.

https://docs.avaloniaui.net/docs/concepts/reactiveui/routing

**Reply by xky0007** [reply]
@timunie Thanks. Maybe it's good to mark outdated at that page to avoid misleading.
